JetLagged/app/build.gradle.kts

```diff
@@ -22,13 +23,22 @@ plugins {
 }
 
 android {
-    compileSdk = libs.versions.compileSdk.get().toInt()
+    compileSdk =
+        libs.versions.compileSdk
+            .get()
+            .toInt()
```

Comment:
This looks a bit overly optimized, are we sure we want this level of formatting?